packages/table/src/common/classes.ts

```diff
@@ -0,0 +1,100 @@
+export const TABLE_LAST_IN_COLUMN = "bp-table-last-in-column";
+export const TABLE_ROUNDED_LAYOUT = "bp-table-rounded-layout";
+export const TABLE_NO_LAYOUT = "bp-table-no-layout";
+export const TABLE_THEAD = "bp-table-thead";
+export const TABLE_COLUMN_HEADER_TR = "bp-table-column-header-tr";
+export const TABLE_DRAGGABLE = "bp-table-draggable";
+export const TABLE_HEADER = "bp-table-header";
+export const TABLE_COLUMN_NAME = "bp-table-column-name";
+export const TABLE_HEADER_CONTENT = "bp-table-header-content";
+export const TABLE_COLUMN_NAME_TEXT = "bp-table-column-name-text";
+export const TABLE_INTERACTION_BAR = "bp-table-interaction-bar";
+export const TABLE_HORIZONTAL_CELL_DIVIDER = "bp-table-horizontal-cell-divider";
+export const TABLE_TH_MENU = "bp-table-th-menu";
+export const TABLE_HEADER_ACTIVE = "bp-table-header-active";
+export const TABLE_HEADER_SELECTED = "bp-table-header-selected";
+export const TABLE_ROW_NAME = "bp-table-row-name";
+export const TABLE_ROW_NAME_TEXT = "bp-table-row-name-text";
+export const TABLE_RESIZE_HANDLE_TARGET = "bp-table-resize-handle-target";
+export const TABLE_DRAGGING = "bp-table-dragging";
+export const TABLE_RESIZE_HORIZONTAL = "bp-table-resize-horizontal";
+export const TABLE_RESIZE_VERTICAL = "bp-table-resize-vertical";
+export const TABLE_RESIZE_HANDLE = "bp-table-resize-handle";
+export const TABLE_RESIZE_SENSOR = "bp-table-resize-sensor";
+export const TABLE_RESIZE_SENSOR_EXPAND = "bp-table-resize-sensor-expand";
+export const TABLE_RESIZE_SENSOR_SHRINK = "bp-table-resize-sensor-shrink";
+export const TABLE_OVERLAY_LAYER = "bp-table-overlay-layer";
+export const TABLE_OVERLAY = "bp-table-overlay";
+export const TABLE_VERTICAL_GUIDE = "bp-table-vertical-guide";
+export const TABLE_HORIZONTAL_GUIDE = "bp-table-horizontal-guide";
+export const TABLE_REGION = "bp-table-region";
+export const TABLE_SELECTION_REGION = "bp-table-selection-region";
+export const TABLE_CONTAINER = "bp-table-container";
+export const TABLE_TOP_CONTAINER = "bp-table-top-container";
+export const TABLE_BOTTOM_CONTAINER = "bp-table-bottom-container";
+export const TABLE_MENU = "bp-table-menu";
+export const TABLE_COLUMN_HEADERS = "bp-table-column-headers";
+export const TABLE_SELECTION_ENABLED = "bp-table-selection-enabled";
+export const TABLE_ROW_HEADERS = "bp-table-row-headers";
+export const TABLE_BODY = "bp-table-body";
+export const TABLE_NO_HORIZONTAL_SCROLL = "bp-table-no-horizontal-scroll";
+export const TABLE_NO_VERTICAL_SCROLL = "bp-table-no-vertical-scroll";
+export const TABLE_BODY_SCROLL_CLIENT = "bp-table-body-scroll-client";
+export const TABLE_RESIZE_GUIDES = "bp-table-resize-guides";
+export const TABLE_BODY_VIRTUAL_CLIENT = "bp-table-body-virtual-client";
+export const TABLE_CELL_CLIENT = "bp-table-cell-client";
+export const TABLE_CELL_GHOST = "bp-table-cell-ghost";
+export const TABLE_CELL_LEDGER_ODD = "bp-table-cell-ledger-odd";
+export const TABLE_CELL_LEDGER_EVEN = "bp-table-cell-ledger-even";
```

Comment:
wow please organize these _somehow_. at the very least alphabetize. at the most, sort them by usefulness, then by alph.